src/status_im/ui/screens/chat/views.cljs

```diff
@@ -300,7 +352,9 @@
                                  (re-frame/dispatch [:set :layout-height (-> e .-nativeEvent .-layout .-height)]))}
         [chat-toolbar current-chat public? modal?]
         [messages-view-animation
-         [messages-view current-chat modal?]]
+         (if platform/desktop?
+           [messages-view-desktop current-chat modal?]
```

Comment:
probably a TODO comment here would be nice

Reply:
added one

Reply:
Thanks!

Reply:
what about other places where flatlist is used? tbh i thought that we make changes in flatlist component itself